MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
+                Format="{Count}",
+                SummaryType=SummaryType.CountAggregate
+            }
+        }
+    });
+
+{% endhighlight %}
+{% endtabs %}
+
+
+![DataGrid with group summary](Images\Group-Summary\ShowSummaryInRow.png)
+
+### Displaying summary in the column
+
+The summary information can be displayed in the column by setting the [DataGridSummaryRow.ShowSummaryInRow]() to `false` and by defining summary columns. To calculate summary based on the column, specify the following properties:
+
+1. [DataGridSummaryColumn.MappingName](): Provides MappingName of the column (Property name of data object) that you want to calculate summary.
+2. [DataGridSummaryColumn.SummaryType](): Provides different built-in summary calculation functions for various types.
+3. [DataGridSummaryColumn.Format](): Provides format string for the summary based on support function name in the specified SummaryType.
+
+Refer to [Formatting Summary](#formatting-summary) section to know more about how to format summary and [Aggregate Types](#aggregate-types) section to know about different Summary Types.
+
+In the following code snippet, summary is defined for `Salary` and `CustomerID` columns:
+
+{% tabs %}
+{% highlight xaml %}
+    <sfgrid:SfDataGrid x:Name="dataGrid" AllowResizingColumn="True"
+                       AutoGenerateColumns="True" 
+                       ColumnSizer="Star">
+       <sfgrid:SfDataGrid.GroupSummaryRows>
+            <sfgrid:DataGridSummaryRow ShowSummaryInRow="False" >
+                <sfgrid:DataGridSummaryRow.SummaryColumns>
+                    <sfgrid:DataGridSummaryColumn Name="Salary" 
+                                                  MappingName="Salary" 
+                                                  Format="{}{Sum}"
+                                                  SummaryType="DoubleAggregate">
+                    </sfgrid:DataGridSummaryColumn>
+                </sfgrid:DataGridSummaryRow.SummaryColumns>
+            </sfgrid:DataGridSummaryRow>
+        </sfgrid:SfDataGrid.GroupSummaryRows>
+    </sfgrid:SfDataGrid>
+
+{% endhighlight %}
+
+{% highlight c# %}
+            this.dataGrid.GroupSummaryRows.Add(new DataGridSummaryRow()
+            {
+                ShowSummaryInRow = false,
+                SummaryColumns = new ObservableCollection<ISummaryColumn>()
+                {
+                    new DataGridSummaryColumn()
+                    {
+                        Name="Salary",
+                        MappingName="Salary",
+                        SummaryType=SummaryType.DoubleAggregate,
+                        Format="{Sum}"
+                    },
+                }
+            });
+{% endhighlight %}
+{% endtabs %}
+
+
+![DataGrid with customized summary for group summary cells](Images\Group-Summary\ShowSummaryInColumn.png)
+
+### Group summary template
+
+The data grid hosts any view(s) inside a group summary for the entire row or for individual columns by loading a template.  
```

Comment:
The data grid allows you to host any view(s) inside a group summary for the entire row or individual columns by loading a template.

Reply:
Modified.